_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
+- Current portals and accounts
+- OneGC
+- API Store
+- Pan-Canadian Trust Framework
+- Some benefits integrated with 3rd parties
+
 ## Businesses
 
+- Able to find and integrate with 50% of ESDC services
+
 ## Non-profits
 
-## IITB
+- Able to find and integrate with 50% of ESDC services
+
+## IITB-ESDC
+
+- IITB is now involved in business discussions and decisions
```

Comment:
Maybe we should be more precise as of currently, the CIO is a member of the "Major Projects and Investment Board (MPIB)".